lms/djangoapps/discussion_api/api.py

```diff
@@ -274,6 +277,7 @@ def get_thread_list(
         "last_activity_at".
     order_direction: The direction in which to sort the threads by. The only
         values are "asc" or "desc". The default is "desc".
+    discussion_id: Content-specific discussion topic ID
```

Comment:
Hmm.. interesting.  I still haven't gotten a strong grasp on the current Discussion terminology.  
The underlying comments service API seems to have a very generic design.  So it may overload multiple concepts into one.  (For example, responses = comments.)  The question is, are "topics" equivalent to "inline discussions" at the outer discussion API layer?

@BenjiLee @aleffert @robrap  What is your take on this?  We want the mobile apps to be able to get all discussion threads filtered by the ID of an inline discussion module.  Can the existing filter, named "topic_id", also apply to inline discussion IDs?  If so, we won't need to change the existing discussion API endpoint.  Otherwise, we'll add a new filter for discussion_ids, but have the underlying implementation equate it to topic_ids.